auto_tests/tests/data_api.js

```diff
@@ -99,4 +99,21 @@ it('testGetRowForXDuplicates', function() {
   assert.equal(5, g.getRowForX(9));
 });
 
+// indexFromSeriesName should return a value even if the series is invisible
+// In 1.1.1, if you request the last set and it's invisible, the method returns undefined.
+it('testIndexFromSetNameOnInvisibleSet', function() {
+  
+  var localOpts = opts;
```

Comment:
This doesn't actually copy anything. If you want to do that there's a method in utils.